packages/utils/feed/queries.ts

```diff
@@ -14,14 +11,17 @@ import {
 } from "../types/feed";
 import { RemoteFileData } from "../types/files";
 
+import { useSigningSocialFeedClient } from "@/hooks/feed/useSocialFeedClient";
+
 export const getAvailableFreePost = async (userId: string) => {
   const [network, userAddress] = parseUserId(userId);
 
   if (!userAddress || network?.kind !== NetworkKind.Cosmos) {
     return;
   }
 
-  const client = await signingSocialFeedClient({
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const client = await useSigningSocialFeedClient({
```

Comment:
this eslint error is pointing at a real bug, if you use a hook conditionally, it will crash the render (the condition is the if with return up